projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -1417,35 +996,6 @@
 			}
 		}
 
-		function loadFullImage( slide ) {
```

Comment:
need to add this back in so we are getting the best quality image for current slide.

Reply:
done